build.sbt

```diff
@@ -836,15 +855,10 @@ lazy val `profiling-utils` = project
       "junit"          % "junit"           % junitVersion   % Test,
       "com.github.sbt" % "junit-interface" % junitIfVersion % Test
     ),
-    modulePath := {
-      JPMSUtils.filterModulesFromUpdate(
-        update.value,
-        Seq(
-          "org.netbeans.api" % "org-netbeans-modules-sampler" % netbeansApiVersion,
-          "org.slf4j"        % "slf4j-api"                    % slf4jVersion
-        ),
-        streams.value.log,
-        shouldContainAll = true
+    moduleDependencies := {
```

Comment:
Same format as `libraryDependencies`. No longer need to call this cryptic `filterModulesFromUpdate` methods ...